modules/ROOT/partials/misc/advcode-shortcuts.adoc

```diff
@@ -1,13 +1,21 @@
-== Advanced Code Editor search and replace keyboard shortcuts
+== Enhanced Code Editor search and replace keyboard shortcuts
 
-Advanced Code Editor provides the following shortcuts within the code dialog.
+Enhanced Code Editor provides the following shortcuts within the code dialog.
 
 [cols=",,",options="header"]
-|===
-|Action |PC |Mac
-|Find |Ctrl+F |Command+F
-|Find next instance |Ctrl+G |Command+G
-|Find previous instance |Shift+Ctrl+G |Shift+Command+G
-|Replace |Ctrl+H |Command+Option+F
-|Replace All |Shift+Ctrl+R |Shift+Command+Option+F
-|===
+|=== 
+| Action | Windows Shortcut | macOS Shortcut
+| Find | Ctrl+F | Command+F
+| Replace | Ctrl+H | Command+Option+F
+| Replace All | Shift+Ctrl+R | Shift+Command+Option+F
```

Comment:
@lorenzo-pomili
Windows: `Chrome` and `FireFox` 
- `Ctrl+H`: is overrided by browser history page
- `Ctrl+Shift+R`: is overrided by browser hard refresh

Reply:
```suggestion
```

Reply:
Removing these commands from the documentation until further testing can be verified.